src/3rdparty/walkontable/test/spec/table.spec.js

```diff
@@ -1213,7 +1586,29 @@ describe('WalkontableTable', () => {
       wt.scrollViewportVertically(10);
       wt.draw();
 
-      expect(wt.wtTable.getFirstRenderedRow()).toBe(3);
+      expect(wt.wtTable.getLastRenderedRow()).toBe(11); // TODO I think this should be 10, investigate
```

Comment:
Did you investigate this behaviour?

Reply:
No, I think that would be beyond the scope of this change. The line in question was changed from performing a wrong assertion into performing a correct assertion of the current code.

Reply:
Alright then. I guess we have to investigate why `ViewportRowsCalculator` returns the incorrect number of rows to render.